src/content/en/updates/2020/06/devtools.md

```diff
@@ -0,0 +1,200 @@
+Paint (LCP), Cumulative Layout Shift (CLS), and Total Blocking Time (TBT). 
+LCP and CLS are 2 of Google's new [Core Web Vitals](https://web.dev/vitals/#core-web-vitals),
+and TBT is a lab measurement proxy for another Core Web Vital, First Input Delay.
+
+The performance score formula has also been reweighted to better reflect the users’
+loading experience.
+
+![New performance metrics in Lighthouse 6.0](/web/updates/images/2020/06/lighthouse.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/67aa7a28ddd7140a03bf6fe16979fc2e040276c1 #}
+
+Chromium issue [#772558](https://crbug.com/772558)
+
+### First Meaningful Paint (FMP) deprecation {: #fmp-deprecation }
+
+[lcp]: https://web.dev/lcp/
+[fmp]: https://web.dev/first-meaningful-paint/
+
+First Meaningful Paint (FMP) is deprecated in Lighthouse 6.0. It has also been removed from the Performance panel. [Largest Contentful Paint][lcp] is the recommended replacement for FMP. See [First Meaningful Paint][fmp] for an explanation of why it was deprecated.
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/cac292283bfa951d65a870b5a46136eff3963d0c #}
+
+Chromium issue [#1096008](https://crbug.com/1096008)
+
+## Support for new JavaScript features {: #javascript }
+
+DevTools now has better support for some of the latest JavaScript language features:
+
+* [Optional chaining](https://v8.dev/features/optional-chaining) syntax autocompletion - property auto-completion in the Console now supports optional chaining syntax, e.g. `name?.` now works in addition to `name.` and `name[`.
+* Syntax highlighting for [private fields](https://v8.dev/features/class-fields#private-class-fields) - private class fields are now properly syntax-highlighted and pretty-printed in the Sources panel.
+* Syntax highlighting for [Nullish coalescing operator](https://v8.dev/features/nullish-coalescing) - DevTools now properly pretty-prints the nullish coalescing operator in the Sources panel.
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend.git/+/80ef20871d541f5e6d789ac2679e1ecd0b2e8328 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/b674ea36ab2fab389e7bc57b6651f1103c0f78d0 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/85ae1392c509c2ea37c89b386c0123350ea53855 #}
+
+Chromium issues [#1083214](https://crbug.com/1083214), [#1073903](https://crbug.com/1073903), [#1083797](https://crbug.com/1083797)
+
+
+## New app shortcut warnings in the Manifest pane {: #app-shortcut-warnings }
+
+[shortcuts]: https://web.dev/app-shortcuts
+
+[App shortcuts][shortcuts] help users quickly start common or recommended tasks within a web app. 
+
+The Manifest pane now shows warnings if:
+
+* the app shortcut icons are smaller than 96x96 pixels
+* the app shortcut icons and manifest icons are not square (as they will be ignored)
+
+![App shortcut warnings](/web/updates/images/2020/06/app-shortcut-warnings.png)
```

Comment:
In general you can make these screenshots look a bit more polished if you use the "how to capture a window" workflow: https://support.apple.com/en-us/HT201361